### PR TITLE
Note that reviewers should mention the id of the commits they reviewed.

### PR DESCRIPTION
**Pull request by pstratem**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -95,6 +95,8 @@ Anyone may participate in peer review which is expressed by comments in the pull
   - Concept ACK means "I agree in the general principle of this pull request";
   - Nit refers to trivial, often non-blocking issues.
 
+Reviewers should include the id of the commits which they reviewed in their comments
```
Comment (luke-jr) [inline CONTRIBUTING.md]: commit hash*

**Comment by dcousens** [comment]
Not that it should be trusted verbatim, but if any changes occur after a comment, then the changed commits will be moved *below* the original comment.

**Comment by pstratem** [comment]
@dcousens yes but without a reference to which commits were ACK'd it's impossible to tell what if anything was changed

for an example of this see #3154

File: CONTRIBUTING.md (hunk 1 above)
Comment (paveljanik) [inline CONTRIBUTING.md]: And a dot at the end of the sentence.

Comment (luke-jr) [inline CONTRIBUTING.md]: How about this dot: ᣟ

**Comment by dcousens** [comment]
:+1:

File: CONTRIBUTING.md (hunk 1 above)
Comment (paveljanik) [inline CONTRIBUTING.md]: I do not see any dot there Luke 8)

**Comment by laanwj** [comment]
Agree that this is useful in some cases, but as long as this is a manual step I don't really want to force it on everyone.

**Comment by pstratem** [comment]
@laanwj sure thus, SHOULD instead of MUST

**Comment by laanwj** [comment]
Ok, concept ACK, agree on the style nits.

**Comment by pstratem** [comment]
@luke-jr is that acceptable?

**Comment by luke-jr** [comment]
Sure

**Comment by fanquake** [comment]
ACK, after updating the commit message. It still mentions id rather than hash.

**Comment by MarcoFalke** [comment]
ACK  7ad0537

**Comment by paveljanik** [comment]
ACK https://github.com/pstratem/bitcoin/commit/7ad0537ee00f5bf792b0156de06a3207e4172009

**Comment by petertodd** [comment]
ACK https://github.com/pstratem/bitcoin/commit/7ad0537ee00f5bf792b0156de06a3207e4172009

**Comment by pstratem** [comment]
@fanquake nit picked

**Comment by fanquake** [comment]
ACK https://github.com/pstratem/bitcoin/commit/e1030dddab11553d2854c1f466e5757d9815bfb8

**Comment by MarcoFalke** [comment]
ACK  e1030dd

**Comment by pstratem** [comment]
another great example of why this is necessary https://github.com/bitcoin/bitcoin/pull/6451

**Pull request merged**